### ISSUE TITLE
Add Organization Type to Org Settings

### ISSUE THREAD

**Issue by aliciabeckett-gov** [CDCgov/prime-simplereport#1187]
We would like to know the type of setting for organizations for 2 reasons:

1. It allows us to do more analyses through Metabase (currently we categorize them by hand as needed) which we report on weekly and use to understand our user base.

2. It allows us to build setting specific features. For example showing school IDs to K12 schools and universities, or offering NHSN uploads to nursing homes.

- The Organization Type field will go on the Organization settings page. It should be set up as a part of the "Create Organization" workflow. 
- It should be a required field for creating an organization

Possible Answers:
- K-12 School
- College/University
- Correctional Facility
- Airport/Transit Station
- Homeless Shelter
- FQHC
- Primary Care / Mental Health Outpatient
- Assisted Living Facility
- Hospital or Clinic
- Urgent Care
- Nursing Home
- Substance Abuse Treatment Center
- Hospice
- Pharmacy
- Other

**TODO**

- [x] Add Organization Type field to Organization Settings page & App database
- [x] Update the Create Organization workflow to add Organization Settings as a required field
- [ ] Sent this data to ReportStream as "Site_of-Care" (see ticket #466)
- [ ] ensure this is set in the account-request/ form and that the signup form still works in a cloud env. Including that the data is still correctly sent to the test Dynamics instance

![image.png](https://images.zenhubusercontent.com/6006fd7f7815d74bc6dbed0a/283e3ac7-29e7-4831-bbb1-5502d7d45776)

**Comment by aliciabeckett-gov** [comment]
This may encompass #466 too

**Comment by sparkle-usds** [comment]
Based on prior conversations I might make these updates...

Airport --> Airport/Transit station
University --> College/University

Potentially add - Childcare or daycare - although we haven't seen this yet and may not so I'd also be fine leaving this out for now.

**Comment by aliciabeckett-gov** [comment]
Ask data hub if they will do the mapping to snowmed for us.

Add work to backfill org type

**Comment by aliciabeckett-gov** [comment]
@aliciabeckett-gov to follow up with @jimduff-usds on mapping.

**Comment by jimduff-usds** [comment]
@Alicia, we can set up a mapping.   However, looking at the official values for site_of_care, we'd have trouble figuring out how to map some of them.  There's no values like School, Airport, Day Care, for example.   Maybe there's some other 'location' code that would work, however, I haven't looked.

Another simple alternative is just to keep it as a text string, and not try to map it to a snomed or other code.

**Comment by jimduff-usds** [comment]
@aliciabeckett-gov , yes we can do this.  Just add the column, and let us know what the column name is, and what the set of possible values is.

**Comment by nickscialli-usds** [comment]
A couple questions as I'm working through this:

- The "Create Organization" workflow in the site admin part of the website was [removed a few days ago](https://github.com/CDCgov/prime-simplereport/pull/1865) so I assume this is no longer part of the ticket, unless I'm missing somewhere else in the app that orgs are created.
- On the sign-up form, we currently have a "Testing facility type" question that has some of the same options as above listed. Is the intent to keep using this field but map it to the new organization type created in this ticket?

![Testing facility type](https://images.zenhubusercontent.com/6025a889a2838a0331f201d8/44b85ebe-39ff-4b1b-9596-6c2e2dce363b)

- The screenshot also shows a text input if "Other" is selected. Is that no longer required?

**Comment by aliciabeckett-gov** [comment]
@nickscialli-usds 

1. > The "Create Organization" workflow in the site admin part of the website was removed a few days >ago so I assume this is no longer part of the ticket, unless I'm missing somewhere else in the app that >orgs are created.

I believe that although it was removed it is still available to engineers. @timothybest-usds @LGFranklin do you know if we need to still support that workflow?


2. >On the sign-up form, we currently have a "Testing facility type" question that has some of the >same options as above listed. Is the intent to keep using this field but map it to the new organization >type created in this ticket?

Yes! That field should be updated with the values in this ticket so that when organizations sign up they have the same options that are in the organization settings page.

3. >The screenshot also shows a text input if "Other" is selected. Is that no longer required?
For other: please keep the "other" option, but going forward we do not need a text box to describe the type of facility.

**Comment by LGFranklin** [comment]
@nickscialli-usds cc: @aliciabeckett-gov @TimBest It will still be available to engineers so we do still want to support it, though the hope is they will rarely need it.

**Comment by nickscialli-usds** [comment]
@aliciabeckett-gov @LGFranklin Thank you! Yes, the GraphQL query for devs to use as necessary will be updated to include the org type so it'll still work for us.

**Comment by nickscialli-usds** [comment]
@aliciabeckett-gov This is going to be blocked on #1833, which is in progress right now. The ReportStream folks rightly want to test this in a lower environment before sending a new field in production.

One option, which I think I'm in favor of, is breaking the "report this field to reportstream" work into a separate ticket so we're not blocked on hooking up our test environments. Thoughts?

**Comment by nickscialli-usds** [comment]
Based on discussion in standup today, I created #1904 to report this data to ReportStream as a separate piece of work so we don't hold this ticket up.

**Comment by aliciabeckett-gov** [comment]
@nickscialli-usds breaking this up into smaller tickets makes sense to me.

**Comment by Rebecca-LM** [comment]
Hi all, I noticed a couple mistakes with capitalization on these so I've fixed them below. Since we use sentence case for almost everything, we should only capitalize the first letter of the first word, unless it comes after a slash (/) or if it's a proper name. We also need to spell out any acronyms on first mention on any page or from to be inclusive of those that are not familiar with certain terms. I've copy/pasted the list below with the corrections. Lmk if anyone has questions. @bmaden-usds 

- K-12 school
- Camp
- College/University
- Correctional facility
- Employer
- Government agency
- Airport/Transit station
- Homeless shelter
- Federally Qualified Health Center (FQHC)
- Primary care/Mental health outpatient
- Assisted living facility
- Hospital or clinic
- Urgent care
- Nursing home
- Substance abuse treatment center
- Hospice
- Pharmacy
- Lab
- Other
